### PR TITLE
klocwork critical issues

### PR DESCRIPTION
**Pull request by kbillore**
Signed-off-by: Shiva Kumara <shiva.kumara.rudrappa@intel.com>
Signed-off-by: Kaushal Billore <kaushal.billore@intel.com>

### CODE REVIEW

**Comment by kbillore** [comment]
@shivasku82 @JeevakaPrabu Please review and approve.

File: camera/MultiCameraApplication/java/com/intel/multicamera/CameraBase.java [hunk 1]
```diff
@@ -203,6 +203,7 @@ public void onClick(View v) {
                             mActivity.sendBroadcast(scanIntent);
                             preView.setImageResource(android.R.color.background_dark);
                             details.setVisibility(View.INVISIBLE);
+
```
Comment (JeevakaPrabu) [inline camera/MultiCameraApplication/java/com/intel/multicamera/CameraBase.java]: blank line not needed

File: camera/MultiCameraApplication/java/com/intel/multicamera/CameraBase.java [hunk 2]
```diff
@@ -404,6 +405,8 @@ private void openCamera(int width, int height) {
             if (map == null) return;
 
             String Key = GetChnagedPrefKey();
+            if(Key == null)
```
Comment (JeevakaPrabu) [inline camera/MultiCameraApplication/java/com/intel/multicamera/CameraBase.java]: introduce a space between if and (

File: camera/MultiCameraApplication/java/com/intel/multicamera/CameraBase.java [hunk 3]
```diff
@@ -404,6 +405,8 @@ private void openCamera(int width, int height) {
...
+		return;
```
Comment (JeevakaPrabu) [inline camera/MultiCameraApplication/java/com/intel/multicamera/CameraBase.java]: 4 space indentation

File: camera/MultiCameraApplication/java/com/intel/multicamera/Utils.java [hunk 4]
```diff
@@ -404,6 +404,12 @@ public static Bitmap loadImageThumbnailFromStream(InputStream stream, int imageW
         bitmap = loadImageThumbnailFromStream(stream, width, height, (int)(dim.x * 0.7f),
                                               (int)(dim.y * 0.7), 0, MAX_PEEK_BITMAP_PIXELS);
 
+        try {
+            stream.close();
+        }catch (Exception e) {
```
Comment (JeevakaPrabu) [inline camera/MultiCameraApplication/java/com/intel/multicamera/Utils.java]: introduce a space between } and catch

File: camera/MultiCameraApplication/java/com/intel/multicamera/CameraBase.java (hunk 2 above)
Comment (kbillore) [reply camera/MultiCameraApplication/java/com/intel/multicamera/CameraBase.java]: resolved.

File: camera/MultiCameraApplication/java/com/intel/multicamera/CameraBase.java (hunk 3 above)
Comment (kbillore) [reply camera/MultiCameraApplication/java/com/intel/multicamera/CameraBase.java]: resolved.

File: camera/MultiCameraApplication/java/com/intel/multicamera/Utils.java (hunk 4 above)
Comment (kbillore) [reply camera/MultiCameraApplication/java/com/intel/multicamera/Utils.java]: resolved.

**Pull request merged**